applications/openshift/master/file_groupowner_ovs_conf_db_aarch64/rule.yml

```
@@ -0,0 +1,37 @@
+documentation_complete: true
+
+platform: ocp4-node and aarch64_arch
```

Strange - this rule isn't even getting reported in the results:

https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/ComplianceAsCode_content/13201/pull-ci-ComplianceAsCode-content-master-e2e-aws-ocp4-cis-node-arm/1904603840137662464

---

Grab the html report from the ARF results and see why this failed.

---

```suggestion
platforms: 
    - ocp4-node
    - aarch64_arch
```

There is an issue with the filter_rules where when the platforms is defined within a single line the comparison will fail thinking that ocp4-node is not part of the platforms, like this:
```
>>> platforms = {'ocp4-node and aarch64_arch'}
>>> "ocp4-node" in platforms
False
```

---

😮‍💨 thanks for the help, that worked!

---

The ovs and sdn filters are implemented in https://github.com/ComplianceAsCode/content/blob/master/products/ocp4/profiles/cis-node-1-7.profile#L26